deployment/ansible/roles/driver.monit/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Download monit
+  get_url: >
+    url=https://mmonit.com/monit/dist/binary/{{ monit_version }}/monit-{{ monit_version }}-linux-x64.tar.gz
```

Comment:
Yeah, the goal was to find a program that will return 0 if a container is running, non-zero otherwise, ideally without piping to `grep` and friends, which is why `docker ps` didn't seem to fit the bill. It does seem like this is sort of a side-effect of `docker top` so I'm open to other suggestions.